src/todoList.js

```diff
@@ -0,0 +1,18 @@
+export default class ToDoList {
+  static list = []
```

Comment:
The task object's values for `completed` should be updated upon user actions. The LIst is empty so please add some items and make sure their completed value becomes true when checked and vice versa.

Reply:
The list is not empty, if I populate this array my elements will be doubled. Also my tasks are already being kept in track of his completion

Reply:
Please watch this video for clarity https://www.loom.com/share/f56a54c5cd604f25b8aad4211f2a3934

Reply:
Kindly populate your empty array with an array of objects that should contain: 
- Methods related to the status updates (completed: true / false).
- It has event listener on the checkbox (change).
- The task object's values for completed is updated upon user actions.
- Update the array of items is stored in local storage, so the user gets the correct list values after the page reloads.